src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -298,6 +298,10 @@ public GHRef createRef(String name, String sha) throws IOException {
     public List<GHRelease> getReleases() throws IOException {
         return listReleases().asList();
     }
+    
+	public GHRelease latestRelease() throws IOException {
```

Comment:
fixed, Thank you

- https://github.com/kohsuke/github-api/pull/343/commits/fd859815b039a84c3dd53e38ff8d41fefc2c2b23?diff=unified